src/components/Login/Login.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import './Login.css';
+import mockUsers from '../MockData/mockusers';
+import { useNavigate } from 'react-router-dom'; 
+
+const Login = ({ onLogin }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [loginError, setLoginError] = useState('');
+  const navigate = useNavigate(); 
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const user = mockUsers.find(
+      (user) => user.username === username && user.password === password
+    );
+    if (user) {
+      onLogin(username, password);
```

Comment:
what is onLogin?  I see the event in App above, but I can't find any docs about it

Reply:
@mbenfowler it's the handleLogin function found in the App component